service/storage_service.cc

```diff
@@ -6200,14 +6200,8 @@ future<> storage_service::await_topology_quiesced() {
         co_return;
     }
 
-    group0_guard guard = co_await _group0->client().start_operation(&_group0_as, raft_timeout{});
-
-    while (_topology_state_machine._topology.is_busy()) {
-        rtlogger.debug("await_topology_quiesced(): topology is busy");
-        release_guard(std::move(guard));
-        co_await _topology_state_machine.event.wait();
-        guard = co_await _group0->client().start_operation(&_group0_as, raft_timeout{});
-    }
+    co_await _group0->group0_server().read_barrier(&_group0_as);
```

Comment:
So the `read_barrier` replaces the guard we get from `start_operation` in the replaced code?